storage/innobase/dict/dict0dd.cc

```diff
@@ -528,7 +528,7 @@ dict_table_t *dd_table_create_on_dd_obj(const dd::Table *dd_table,
     if (dd_col->is_virtual()) {
       n_v_cols++;
       if (dd_col->is_array()) {
-         n_m_v_cols++;
```

Comment:
This might be your editor, which is adjusting the whole file and looking for formatting issues. Please disable it.

Reply:
yes, I think it is clang formatter doing this automatically. I configured it for PS should not we use it for PXB ?